### DISCUSSION TITLE
It's alive !

### DISCUSSION THREAD

**Discussion by Yago** [webmardi/webmardi.ch#43] [Announcements]
Hi @CamilleLetang, @WengerK, @gperriard and @nlsnmr ! 🤗

The new website is alive ! 🚀  Everything should be working as expected, but if you see anything, simply answer to this thread and I will fix it ASAP 😉

Otherwise, I still have to improve the “rebuild-on-demand” system; I will find a solution still this week and in the best-case scenario, I'm going to add a button in Notion to republish the website. I'll keep you in touch.

I hope it will satisfy you,
Have a nice evening 😄 🤙

**Comment by gperriard** [comment]
Hello Yann,

Thank you very much for your work.
It's really good.

Regards,
Geoffroy

**Comment by Yago** [comment]
Here we are, the deploy process is up and running. You'll find everything in the “Website tools” Notion page. Don't hesitate if you have any question 😉

<img width="1136" alt="CleanShot 2021-11-10 at 18 02 36@2x" src="https://user-images.githubusercontent.com/2025924/141158762-4ad430d5-8e94-4b8f-9288-90973e89db67.png">
